docs/sources/project/work-issue.md

```diff
@@ -109,7 +109,7 @@ Follow this workflow as you work:
 
 9. Push your change to your repository.
 
-        $ git push origin
+        $ git push origin 11038-fix-rhel-link
```

Comment:
I always thought explicit is better but that could just be me. One advantage of explicitly mentioning the branch is that the output shown in the docs has better match with what the users are likely to see in their invocation.